layers/core_checks/cc_cmd_buffer_dynamic.cpp

```
@@ -2266,8 +2266,8 @@ bool CoreChecks::PreCallValidateCmdSetAttachmentFeedbackLoopEnableEXT(VkCommandB
                          string_VkImageAspectFlags(aspectMask).c_str());
     }
 
-    if (aspectMask != VK_IMAGE_ASPECT_NONE && aspectMask != VK_IMAGE_ASPECT_COLOR_BIT && aspectMask != VK_IMAGE_ASPECT_DEPTH_BIT &&
-        aspectMask != VK_IMAGE_ASPECT_STENCIL_BIT) {
+    if ((aspectMask &
+         ~(VK_IMAGE_ASPECT_NONE | VK_IMAGE_ASPECT_COLOR_BIT | VK_IMAGE_ASPECT_DEPTH_BIT | VK_IMAGE_ASPECT_STENCIL_BIT)) > 0) {
```

nit: `!= 0` for comparison to sleep better, it's tricky business with signed/unsigned types . Enum values are by default signed, aspect mask is unsigned and then it goes beyond my understanding.